src/utilities/codemirror/block.ts

```diff
@@ -0,0 +1,10 @@
+import { Decoration } from '@codemirror/view';
```

Comment:
Do you mind moving this to a `/codemirror/themes` directory and renaming it to `blockTheme.ts`? Potentially other non-sequence codemirror editors could benefit from this theme as well.

Reply:
addressed in https://github.com/NASA-AMMOS/aerie-ui/pull/1472/commits/710bf93e74e056590a21397eac630292b7716142